packages/calypso-e2e/src/lib/blocks/block-flows/all-form-fields.ts

```diff
@@ -52,32 +59,43 @@ export class AllFormFieldsFlow implements BlockFlow {
 			[ 'Date Picker', 'Add label…' ],
 			[ 'Phone Number Field', 'Add label…' ],
 			[ 'Multi-line Text Field', 'Add label…' ],
-			[ 'Checkbox', 'Add label…' ],
+			[ 'Checkbox', isRefactor ? 'Add option…' : 'Add label…' ],
```

Comment:
This one might be a bug in my branch. 'Add label' seems like it'd be slightly better for a checkbox.

I'll look into it.